id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoProxyServiceImpl.java

```diff
@@ -245,6 +245,8 @@ public class IdRepoProxyServiceImpl implements IdRepoService<IdRequestDTO, IdRes
 	@Autowired
 	private BiometricExtractionService biometricExtractionService; 
 
+	@Value("${mosip.idrepo.update-enabled:false}")
+	private boolean isUpdateEnabled;

```

Comment:
This fix to handle the case of update is a different bug, can you also include the respective JIRA id in the PR title ?